packages/core/parcel/src/cli.js

```diff
@@ -270,6 +270,7 @@ async function normalizeOptions(command): Promise<InitialParcelOptions> {
   let nodeEnv;
   if (command.name() === 'build') {
     nodeEnv = initialNodeEnv || 'production';
+    command.autoinstall = false;
```

Comment:
I would say this should rather be in `@parcel/core`'s `resolveOptions`

Reply:
Are you sure? Don't we want to allow this in the api?

Reply:
Just changed the default to false when using the API, this way anyone can still enable it when using the API but only explicitly

Reply:
I was thinking of `autoinstall: initialOptions.autoinstall ?? mode !== 'production',` but this is fine as well